### DISCUSSION TITLE
I made my own did:key creator

### DISCUSSION THREAD

**Discussion by bshambaugh** [ucan-wg/spec#92] [Show & Tell]
This is my excuse to see how this works. It turns out I baked my own did:key creator:

https://github.com/bshambaugh/did-key-creator

**Comment by bshambaugh** [comment]
Interesting, I found your implementation: 

https://github.com/ucan-wg/ts-ucan/blob/6fad15027523846a2a427ed2e74f34a699cfd79d/packages/default-plugins/src/p256/crypto.ts 

https://github.com/ucan-wg/ts-ucan/blob/6fad15027523846a2a427ed2e74f34a699cfd79d/packages/default-plugins/src/util.ts

I guess we can close this if you want.

**Comment by bmann** [comment]
Hey @bshambaugh — this is a GH Discussion. Totally appropriate to share and promote implementations like you’ve done. Thanks for sharing!

**Reply by bshambaugh** [reply]
Okay, while I am at it here is resolving for the did:key. https://github.com/ceramicnetwork/js-ceramic/tree/develop/packages/key-did-resolver/src . I built all of the NIST stuff. In the future, I may break this stuff out into a different package so I can export some of the utility functions.
